### PR TITLE
adds help command

### PR DESCRIPTION
**Pull request by anistarafdar**
consider adding an ls command, and maybe add command line arguments to main to run stuff before launching the shell if you want, for example "dash help" runs help

hopefully my addition doesnt break the program, i didnt bother compiling to test but it looked fine to me

ive heard that the best practice for using if statements in C is to "always use the { } braces even when they are not necessary", since it makes changing code around it and even reading it easier, imo

### CODE REVIEW

**Comment by anistarafdar** [comment]
https://github.com/chocolate-doom/chocolate-doom/blob/master/HACKING.md

C coding style guideline that i find helpful

File: dash.c [hunk 1]
```diff
@@ -129,21 +129,34 @@ void execute(char** command) {
...
             printf("evaluateCommand: cd failed\n");
             exit(1);
         }
-        return;
     }
 
+    // Check if command is cd; if so, change directory.
+    if (strcmp(command[0], "help") == 0) {	   
+	    puts("\ndash: \"deadass\" shell"
+            "\nGerardo Torres (github.com/gtr)y"
```
Comment (gtr) [inline dash.c]: i think there's a typo extra "y"

**Review by gtr — CHANGES_REQUESTED** [review]
looks good! i'd just fix the typo on line 151. thank you.

**Comment by gtr** [comment]
thanks for your pull request!

> consider adding an ls command, and maybe add command line arguments to main to run stuff before launching the shell if you want, for example "dash help" runs help

the `ls` command actually already works it's delegated to a child process to run separately using `execvp()` on line 110. the reason why `ls` doesn't appear with `cd` and `exit` is because the former are built-in commands into dash which can't be run with `execvp()` using C.

**Review by gtr — APPROVED** [review]

**Comment by gtr** [comment]
thanks!

**Pull request merged**